MyLibrary.Domain.Integration.Tests/BookRepositoryTest.cs

```diff
@@ -0,0 +1,72 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using MyLibrary.Domain.Entities;
+
+namespace MyLibrary.Domain.Integration.Tests
+{
+    [TestClass]
+    public class BookRepositoryTest
+    {
+        private const string CONNECTIONSTRING = "lalala";
+
+        private const string DATABASE_NAME = "";
+        private IBookRepository _bookRepository;
+        [TestInitialize]
+        public void setup()
+        {
+
+        }
+        [TestCleanup]
+        public void Cleanup()
+        {
+
+        }
+
+        [TestMethod]
+        public void GetAll_ReturnsAllBooks()
+        {
+            var result = _bookRepository.GetAll();
+            Assert.AreNotEqual(null, result);
+        }
+        [TestMethod]
+        public void GetOne_ReturnsBookByTitle()
+        {
+            var result = _bookRepository.GetOne("Title");
```

Comment:
It seems odd to get a single book by title, as two books can have the same title, i think maybe having an optional filter on the GetAllBooks() for title to get all book with the same title would be more usefully and you can change this GetOne to return book by Id instead